core/internal/gethwrappers/extract_contract_details.go

```diff
@@ -84,3 +85,9 @@ func ExtractContractDetails(beltArtifactPath string) (*ContractDetails, error) {
 var binarySuffixRegexp = regexp.MustCompile(
 	"^a264697066735822[[:xdigit:]]{68}64736f6c6343[[:xdigit:]]{6}0033$",
 ).MatchString
+
+// constantBinaryMetadataSuffix is a constant stand-in for the metadata suffix
+// which the EVM expects (and in some cases, it seems, requires) to find at the
+// end of the binary object representing a contract under deployment.
+var constantBinaryMetadataSuffix = "a264697066735822" +
```

Comment:
Bizarre... where does this come from out of interest?

Reply:
I should have included a link to this in the comment. I'll do it in a later PR. https://solidity.readthedocs.io/en/v0.5.3/metadata.html